torch/testing/_internal/common_device_type.py

```diff
@@ -1139,6 +1141,18 @@ def _parametrize_test(self, test, generic_cls, device_cls):
                 # Construct parameter kwargs to pass to the test.
                 param_kwargs = {"op": op}
                 _update_param_kwargs(param_kwargs, "dtype", dtype)
+                # Filter sample rules to only apply those that match the op.
```

Comment:
```suggestion
                # Filter sample rules to only those that apply to the op info
```
would saying op info be more precise here?